app/assets/javascripts/discourse/app/components/modal/bulk-topic-actions.gjs

```diff
@@ -41,8 +43,8 @@ export default class BulkTopicActions extends Component {
       return this._processChunks(operation);
     } catch {
       this.dialog.alert(i18n("generic_error"));
+      this.errors = true;
```

Comment:
What happens if there is an error in the actual modal? We should probably reset the loading state, form and `this.errors` right?

Reply:
Uhh good question. Let me see if I can find a good way to test this.

Reply:
https://github.com/discourse/discourse/assets/1490496/bdf2c179-e2b9-48ae-b8b3-9336f1f8a4be

Not really sure how I can do this with a system test. To simulate an error I just edited the controller and forced it to raise an error every time.

Reply:
All good, I just wanted to see what happened if there was an error and that it was handled correctly :) I am guessing in reality the errors should be rare